adapters/handlers/rest/clusterapi/schema_version.go

```diff
@@ -21,7 +21,7 @@ import (
 func extractSchemaVersionFromUrlQuery(values url.Values) uint64 {
 	var schemaVersion uint64
 	if v := values.Get(replica.SchemaVersionKey); v != "" {
-		if vAsUint64, err := strconv.ParseUint(v, 10, 64); err != nil {
+		if vAsUint64, err := strconv.ParseUint(v, 10, 64); err == nil {
```

Comment:
**thought (non-blocking):** given that default case will always be 0 

what do you think if we do something like that 
```
func extractSchemaVersionFromUrlQuery(values url.Values) uint64 {
	schemaVersion, _ := strconv.ParseUint(values.Get(replica.SchemaVersionKey), 10, 64)
	return schemaVersion
}
```

Reply:
I agree, good idea